aspnetcore/blazor/host-and-deploy/server.md

```diff
@@ -62,91 +62,103 @@ Recommendations for global deployments to geographical data centers:
 
 * Deploy the app to the regions where most of the users reside.
 * Take into consideration the increased latency for traffic across continents. To control the appearance of the reconnection UI, see <xref:blazor/fundamentals/signalr#control-when-the-reconnection-ui-appears>.
-* For Azure hosting, use the [Azure SignalR Service](#azure-signalr-service).
+* For Azure hosting outside of Azure App Service, use the [Azure SignalR Service](#azure-signalr-service).
```

Comment:
We want to recommend the Azure SignalR Service only when it's needed. We know it's not needed on Azure App Service and Azure Container apps, but there may also be other Blazor hosting environments that don't benefit from it.

```suggestion
* Consider using the [Azure SignalR Service](#azure-signalr-service).
```